schema.graphql

```diff
@@ -2509,6 +2513,9 @@ type Stay22Hotel implements HotelInterface {
 
   """The star rating of the hotel."""
   rating: HotelRating
+
+  """All ameneties/facilities available in the hotel"""
```

Comment:
>  we run out of synonyms and the field has type the old type

This is better than creating BC and break the app for current users IMO. 

It is a new type, the old type was a connection, which is useless for this field, the new type is a list.

Reply:
I mean, absolutely. 

However, would it make sense to copy the type `HotelFacility` as `HotelAmenity`, deprecate `HotelFacility` so that there is better coupling in the naming of the types and the fields?

Reply:
I think this discussion belongs to

[gitlab PR](https://gitlab.skypicker.com/graphql/graphql/merge_requests/417/diffs#cf086f4561ba749415abdebf71ef60de42160864_92_97)

Reply:
Commented there